test/WebApiShimFw46.FunctionalTests/project.json

```diff
@@ -11,7 +11,8 @@
     "Microsoft.Framework.ConfigurationModel.Json": "1.0.0-beta4",
     "Microsoft.AspNet.Mvc.WebApiCompatShim": "6.0.0-beta4",
     "FunctionalTestUtils": "1.0.0-*",
-    "xunit.runner.aspnet": "2.0.0-aspnet-beta4-*"
+    "xunit.runner.aspnet": "2.0.0-aspnet-beta4-*",
+    "Microsoft.ApplicationInsights": "0.16.1-build00185"
```

Comment:
Alphabetizing lists like this helps to minimize merge conflicts and errors that occur when hidden conflicts are automatically merged by Git. The problem gets worse as the number of items increases.